ts/features/bonus/cgn/screens/CgnOtpScreen.tsx

```diff
@@ -0,0 +1,125 @@
+import useOtpNotWorkingBottomSheet from "../components/otp/OtpNotWorking";
+import { navigateToCgnMerchantsList } from "../navigation/actions";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const getReadableTTL = (
+  otp: RemoteValue<Otp, NetworkError>
+): string | undefined => {
+  if (isReady(otp)) {
+    return otp.value.ttl > 60
+      ? I18n.t("bonus.cgn.otp.code.minutes", {
+          defaultValue: I18n.t("bonus.cgn.otp.code.minutes.other", {
+            minutes: Math.round(otp.value.ttl / 60)
+          }),
+          count: Math.round(otp.value.ttl / 60),
+          minutes: Math.round(otp.value.ttl / 60)
+        })
+      : I18n.t("bonus.cgn.otp.code.seconds", {
+          defaultValue: I18n.t("bonus.cgn.otp.code.seconds.other", {
+            seconds: otp.value.ttl
+          }),
+          count: otp.value.ttl,
+          seconds: otp.value.ttl
+        });
+  }
+  return undefined;
+};
+
+const CgnOtpScreen: React.FunctionComponent<Props> = (props: Props) => {
+  useActionOnFocus(props.generateOtp);
+
+  const readableTTL = getReadableTTL(props.otp);
+
+  const { present } = useOtpNotWorkingBottomSheet(props.navigateToMerchants);
+
+  return (
+    <LoadingSpinnerOverlay isLoading={isLoading(props.otp)}>
+      <BaseScreenComponent goBack contextualHelp={emptyContextualHelp}>
+        <SafeAreaView style={IOStyles.flex}>
+          <ScrollView style={[IOStyles.flex]} bounces={false}>
+            <View style={IOStyles.horizontalContentPadding}>
+              <H1>{"Usa codice sconto"}</H1>
+              {isReady(props.otp) && (
+                <OtpCodeComponent
+                  otp={props.otp.value}
+                  onEnd={props.generateOtp}
+                  progressConfig={{
+                    startPercentage: 100,
+                    endPercentage: 0
+                  }}
+                />
+              )}
+              <View spacer extralarge />
+              {readableTTL && (
+                <Body>{`${I18n.t(
+                  "bonus.cgn.otp.screen.text1"
+                )} ${readableTTL}`}</Body>
+              )}
+              <View spacer large />
+              <Body>
+                <H4 weight={"SemiBold"} color={"bluegreyDark"}>
+                  {I18n.t("global.genericAlert")}
+                </H4>
+                {` ${I18n.t("bonus.cgn.otp.screen.text2")}`}
+              </Body>
+              <View spacer extralarge />
+              <Link style={{ textAlign: "center" }} onPress={present}>
+                {I18n.t("bonus.cgn.otp.screen.link")}
+              </Link>
+            </View>
+          </ScrollView>
+          <FooterWithButtons
+            type={"SingleButton"}
+            leftButton={cancelButtonProps(
+              () =>
+                isReady(props.otp)
+                  ? clipboardSetStringWithFeedback(props.otp.value.code)
+                  : nullType,
+              "Copia codice",
```

Comment:
Use traslation

Reply:
I created the entry in the locales files but missed to replace it!

Adding with other fixes